*** tests/A-06-05-M-ok.out	2022-10-02 09:40:26.980977100 +0100
--- tests/A-06-05-M-ok.outhyp	2022-10-31 15:14:58.915707200 +0000
***************
*** 14,20 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
--- 14,21 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: A recepção de mensagens já está activa
! GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
***************
*** 34,40 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
--- 35,42 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: A recepção de mensagens já está activa
! GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
***************
*** 54,60 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
--- 56,63 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: A recepção de mensagens já está activa
! GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
***************
*** 74,80 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
--- 77,84 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: A recepção de mensagens já está activa
! GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
***************
*** 94,100 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
--- 98,105 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: A recepção de mensagens já está activa
! GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
***************
*** 114,120 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
--- 119,126 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: A recepção de mensagens já está activa
! GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
***************
*** 134,140 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
--- 140,147 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: A recepção de mensagens já está activa
! GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
  3 - Registar cliente
