*** tests/A-11-07-M-ok.out	2022-10-14 11:51:59.201435000 +0100
--- tests/A-11-07-M-ok.outhyp	2022-10-31 15:15:02.235423100 +0000
***************
*** 23,33 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Mensagem de texto a enviar: Enviar comunicação de texto: Operação inválida: O terminal '969090' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
  4 - Adicionar amigo
  5 - Retirar amigo
  6 - Pagamento
--- 23,57 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
+ 4 - Adicionar amigo
+ 5 - Retirar amigo
+ 6 - Pagamento
+ 7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
+ 11 - Mostrar comunicação em curso
+ 0 - Sair
+ Escolha uma opção: OPERAÇÕES DO TERMINAL
+ 1 - Ligar
+ 2 - Colocar no silêncio
+ 3 - Desligar
  4 - Adicionar amigo
  5 - Retirar amigo
  6 - Pagamento
