*** tests/A-12-11-M-ok.out	2022-10-14 17:59:43.183595500 +0100
--- tests/A-12-11-M-ok.outhyp	2022-10-31 15:15:03.525938600 +0000
***************
*** 23,29 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 23,29 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 31,40 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: VIDEO|1|969011|969014|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 31,41 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 43,49 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
--- 44,64 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
