*** tests/A-14-03-M-ok.out	2022-10-21 12:18:41.629846700 +0100
--- tests/A-14-03-M-ok.outhyp	2022-10-31 15:15:04.698474900 +0000
***************
*** 23,29 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 23,29 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 31,50 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|BUSY|0|0
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|BUSY|0|0
  FANCY|969013|cli001|SILENCE|0|0
  FANCY|969014|cli002|IDLE|0|0
  FANCY|969015|cli003|IDLE|0|0
--- 31,65 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: Opção inválida!
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|IDLE|0|0
  FANCY|969013|cli001|SILENCE|0|0
  FANCY|969014|cli002|IDLE|0|0
  FANCY|969015|cli003|IDLE|0|0
***************
*** 61,70 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 76,86 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 77,84 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Não existe nenhuma comunicação em curso.
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 93,111 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 95,101 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|IDLE|0|20
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
--- 122,128 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
