*** tests/A-19-01-M-ok.out	2022-10-27 17:06:52.276590800 +0100
--- tests/A-19-01-M-ok.outhyp	2022-10-31 15:15:08.644539000 +0000
***************
*** 33,38 ****
--- 33,40 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: OPERAÇÕES DO TERMINAL
***************
*** 53,59 ****
  3 - Operações de terminal
  0 - Sair
  Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|IDLE|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
--- 55,61 ----
  3 - Operações de terminal
  0 - Sair
  Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
