*** tests/A-19-07-M-ok.out	2022-10-27 17:16:53.966389900 +0100
--- tests/A-19-07-M-ok.outhyp	2022-10-31 15:15:09.301248600 +0000
***************
*** 33,42 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: O terminal já estava desligado.
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 33,43 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 44,49 ****
--- 45,52 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
***************
*** 59,68 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: O terminal já estava desligado.
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 62,72 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 70,75 ****
--- 74,81 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
