*** tests/A-19-06-M-ok.out	2022-10-27 17:14:52.121472100 +0100
--- tests/A-19-06-M-ok.outhyp	2022-10-31 15:15:09.184914600 +0000
***************
*** 45,50 ****
--- 45,52 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
***************
*** 72,77 ****
--- 74,81 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
***************
*** 79,91 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|OFF|0|0
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
  FANCY|969011|cli001|IDLE|0|0
! FANCY|969013|cli001|OFF|0|0
  FANCY|969014|cli002|IDLE|0|0
  FANCY|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
--- 83,95 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
  FANCY|969011|cli001|IDLE|0|0
! FANCY|969013|cli001|SILENCE|0|0
  FANCY|969014|cli002|IDLE|0|0
  FANCY|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
